go.mod

```diff
@@ -4,32 +4,38 @@ go 1.14
 
 require (
 	github.com/Bose/go-gin-logrus v1.0.3
+	github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6 // indirect
```

Comment:
I know it's an indirect dependency, but apparently this is deprecated, do you know where the dependency comes from?

Reply:
Actually there are quite more dependencies, is it from the common packages?

Reply:
hmm some strange things happens I'm not sure but I think `go` try to check for matching dependencies automatically when it does not find in the `go.mod`
but `go mod tidy` fix this issue, you should have less dependencies now :)
good catch 👍 
I will check the `server` repository to be ensure